modules/tensor_mechanics/test/tests/dynamics/time_integration/tests

```diff
@@ -53,14 +49,24 @@
     input = 'hht_test_ti.i'
     exodiff = 'hht_test_out.e'
     abs_zero = 1e-09
-    compiler = 'GCC CLANG'
 
     requirement = "The mechanics system shall accurately predict the dynamic response of a linear "
                   "elastic system using Hilber-Hughes-Taylor (HHT) time integration when velocity "
                   "and acceleration of the system are calculated using the Newmark-Beta time "
                   "integrator."
     design = "Dynamics.md"
     issues = "#12185"
-    prereq = "hht"
+  []
+
+  [ad_newmark_action]
+    type = 'Exodiff'
+    input = 'ad_newmark_action.i'
```

Comment:
Seems like this input file should be almost identical to `newmark_action.i` -- why not use cli_args?